src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl

```diff
@@ -70,8 +70,7 @@ task modify_file_gcs {
     Boolean ready
     String file_path_raw
     command {
-        echo "override" > tmp
-        gsutil cp tmp ${file_path_raw}
+        echo "override" | gsutil cp - ${file_path_raw}
```

Comment:
Perhaps when that bug is fixed, you can submit another PR to put it back later? 😉